src/styles/typography/index.md.njk

```diff
@@ -102,6 +102,14 @@ You don’t need to use a lead-in line for numbered lists. Items in a numbered l
 
 {{ example({group: 'styles', item: 'typography', example: 'list-number', html: true, open: true}) }}
 
+## Section break &lt;hr&gt;
+
+To create a theamatic break between sections of content you can use the `govuk-section-break` classes on an `<hr>` element. `govuk-section-break` has class based modifiers for different size margins.
```

Comment:
Change:

You can use the `govuk-section-break` classes on an `<hr>` element to create a thematic break between sections of content. `govuk-section-break` has class-based modifiers for different size margins.